Formula/tweak.rb

```diff
@@ -0,0 +1,20 @@
+class Tweak < Formula
+  desc "Command-line, ncurses library based hex editor"
+  homepage "https://www.chiark.greenend.org.uk/~sgtatham/tweak/"
+  url "https://www.chiark.greenend.org.uk/~sgtatham/tweak/tweak-3.02.tar.gz"
+  sha256 "5b4c19b1bf8734d1623e723644b8da58150b882efa9f23bbe797c3922f295a1a"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "make"
+    bin.install "tweak"
+    man1.install "tweak.1"
+    doc.install "btree.html"
+    prefix.install_metafiles
+  end
+
+  test do
+    system bin/"tweak", "--help"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook).

In most cases, a good test would involve running a simple test case: run `#{bin}/foo input.txt`.
- Then you can check that the output is as expected (with `assert_equal` or `assert_match` on the output of `shell_output`)
- You can also check that an output file was created, if that is expected: `assert_predicate testpath/"output.txt", :exist?`

Some advice for specific cases:
- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
tweak is used to view hex files ... not sure what test case is possible, unless there's a file that I know to be the same on all boxes that tweak can be brew installed onto?

or maybe do you want me to do a test to make sure the --help shows exactly the right text by comparing it with the expected --help output?

Reply:
You can create a file, then you know its content :)

Reply:
tried using tweak -D which outputs the default .tweakrc file to stdout ...
alternatively could ... maybe write that to a file? then using tweak to read it? but then couldn't figure out how to compare the contents of an emacs window with asserts

also is it ok to use head?